gen/definitions/snmp_server_vrf_host.yaml

```diff
@@ -0,0 +1,10 @@
+---
+name: SNMP Server VRF Host
+path: Cisco-IOS-XR-um-snmp-server-cfg:snmp-server/vrfs/vrf[vrf-name-%s]/hosts/host[address=%s]
+doc_category: Management
+attributes:
+  - yang_name: vrf-name
+    example: 11.11.11.11
+  - yang_name: traps/unencrypted/unencrypted-string/version/v3/security-level
```

Comment:
`unencrypted-string` is a list so this must become a nested list. We also need to add `community-string` as an attribute of that list.